From e39bf764f67cf42273da22b615b84a08c0c70965 Mon Sep 17 00:00:00 2001
From: Takeshi KOMIYA <i.tkomiya@gmail.com>
Date: Mon, 15 May 2023 11:21:54 +0900
Subject: [PATCH] spec: Use fileutils for testing Solargraph::Rbs::StdlibMap

The "set" has been merged into the Ruby Core and RBS has been drop
support since v3.0.  So current spec implementation will not work with
RBS-3.x.
---
 spec/rbs_map/stdlib_map_spec.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/spec/rbs_map/stdlib_map_spec.rb b/spec/rbs_map/stdlib_map_spec.rb
index 80734530..8a334765 100644
--- a/spec/rbs_map/stdlib_map_spec.rb
+++ b/spec/rbs_map/stdlib_map_spec.rb
@@ -1,7 +1,7 @@
 describe Solargraph::RbsMap::StdlibMap do
   it "finds stdlib require paths" do
-    rbs_map = Solargraph::RbsMap::StdlibMap.load('set')
-    pin = rbs_map.path_pin('Set#add')
+    rbs_map = Solargraph::RbsMap::StdlibMap.load('fileutils')
+    pin = rbs_map.path_pin('FileUtils#chdir')
     expect(pin).to be
   end
 
